.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on:
+  workflow_dispatch:  # Keep manual trigger
+  schedule:
+    - cron: "0 0 * * *"  # Run every day at midnight
```

Comment:
To Use

1. Nightly triggers and make a <VERSION>rc<DATE> package to (e.g. https://test.pypi.org/project/llama-models/0.0.63rc20250109/)

2. Manual trigger make a <VERSION> package to test.pypi